Linux-How-To-Install.md

```diff
@@ -0,0 +1,13 @@
+# How to pick a Linux distribution and install
```

Comment:
I don't have a problem with that, should I make that change while request is still open?